src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -1435,8 +1445,14 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                     });
                 }
             }
+
+            var description = (disable) ? "disable" : "enable";
+            LogTaskToAudit(description + $"-{typeStr}Task", key, configuration: null);
```

Comment:
```suggestion
            LogTaskToAudit(description + $"-{typeStr}-Task", key, configuration: null);
```

we want to show the `taskName` here as well, maybe we can construct the configuration here by ourself?

Reply:
Done